react-tic-tac-toe/src/app/components/App/login/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import userService from '../../../services/usersService';
+import { changeLogged } from '../../../redux/actions/login';
+
+import MyForm from './layout';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: false
+    };
+  }
+
+  handleSubmit = values => {
+    userService
+      .login(values.email, values.password)
+      .then(token => {
+        localStorage.setItem('token', token);
+        this.props.changeLogged(true);
+      })
+      .catch(() => {
+        this.setState({
+          error: true
+        });
+      });
+  };
+
+  render() {
+    if (this.props.logged) {
+      return <Redirect to="/game" />;
+    }
+    if (this.state.error) {
+      return <MyForm onSubmit={this.handleSubmit} loginError="true" />;
+    }
+
+    return <MyForm onSubmit={this.handleSubmit} />;
```

Comment:
If you only are changing one prop you dont need this if
You can pass `loginError={this.state.error}`